MM2RandoLib/Enums/EInstruction.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MM2Randomizer.Enums
+{
+    public enum EInstruction
+    {
+        // We can add more later if useful
+        RTS = 0x60,
+        NOP = 0xEA,
```

Comment:
I think this is okay for now, because it was already written like this, but I am not particularly keen on using an enum in the place of constants.

Reply:
I don't really know C#. What would you have done instead?

Reply:
Well, I have two ideas. One would be to refactor the instructions and address enums into their own classes with these being constants, to avoid the casting. Or, the patch class could have a method that takes these special case enums.  But, again, this is not really a problem with this PR, just something to address down the road.